components/io_expander/esp_io_expander/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+idf_component_register(SRCS "esp_io_expander.c" INCLUDE_DIRS "include" REQUIRES "driver")
```

Comment:
Does this component really require `driver`? I don't see any includes from `driver` here....

Reply:
You're right, it's a mistake. Fixed [here](https://github.com/espressif/esp-bsp/pull/86/commits/ca919496b317741e0e9ac3e8e858dbb289b5e385#diff-8d4b72fb8be6e23a10ef6e5fb3f6f5a6157207e980332eb46f673d2283f2daf9L1-R1).